src/pages/painting/[slug].js

```diff
@@ -47,10 +69,16 @@ export async function getStaticProps({ params }) {
       },
     };
   }
+
+  const recommendedPaintings = removeCurrentPaintingFromRecommendedList(
```

Comment:
How come you are calling this function twice? The return value to props is the exact same.